cmd/tempo-query/tempo/plugin.go

```diff
@@ -53,24 +53,38 @@ var tlsVersions = map[string]uint16{
 }
 
 type Backend struct {
-	tempoBackend    string
-	tlsEnabled      bool
-	tls             tlsCfg.ClientConfig
-	httpClient      *http.Client
-	tenantHeaderKey string
+	tempoBackend          string
+	tlsEnabled            bool
+	tls                   tlsCfg.ClientConfig
+	httpClient            *http.Client
+	tenantHeaderKey       string
+	QueryServicesDuration *time.Duration
```

Comment:
since this is all tag values and not just services i'd adjust the name to reflect that.

Reply:
well, not really, this is only used for fetch service list.

Usually that list only contains services that are in between the 15 minutes (or the defined complete_block_timeout) but now that we can get it from the backend we can extend that time.

Reply:
i see 👍